test/py2ju.jl

```diff
@@ -0,0 +1,21 @@
+using CDSAPI
```

Comment:
No need to import the modules in the `test/foo.jl` files. The modules are imported in `runtests.jl` once. Could you please remove them from the `py2ju.jl` and `retrieve.jl` test files?

Reply:
Done